pkg/policy/api/selector.go

```diff
@@ -386,16 +388,6 @@ func (n *EndpointSelector) Sanitize() error {
 // EndpointSelectorSlice is a slice of EndpointSelectors that can be sorted.
 type EndpointSelectorSlice []EndpointSelector
 
-func (s EndpointSelectorSlice) Len() int      { return len(s) }
-func (s EndpointSelectorSlice) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
-
-func (s EndpointSelectorSlice) Less(i, j int) bool {
-	strI := s[i].LabelSelectorString()
-	strJ := s[j].LabelSelectorString()
-
-	return strings.Compare(strI, strJ) < 0
-}
-
```

Comment:
I'm surprised we can delete these, don't we need to sort the slice for reliable output somewhere in testing code?